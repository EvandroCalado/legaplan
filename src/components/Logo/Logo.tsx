import Image from 'next/image';
import logo from '../../../public/logo.svg';

export const Logo = () => {
  return (
    <Image
      src={logo}
      alt='logo'
      width={150}
      height={36}
      className='logo'
      priority
    />
  );
};
