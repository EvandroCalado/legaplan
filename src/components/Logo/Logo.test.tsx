import { render, screen } from "@testing-library/react";
import { Logo } from "./Logo";

describe("<Logo />", () => {
  it("should render a logo", () => {
    const { container } = render(<Logo />);

    expect(screen.getByRole("img")).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
