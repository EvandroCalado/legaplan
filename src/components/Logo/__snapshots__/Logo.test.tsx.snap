// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Logo /> > should render a logo 1`] = `
<div>
  <img
    alt="logo"
    class="logo"
    data-nimg="1"
    decoding="async"
    fetchpriority="high"
    height="36"
    src="/Logo.svg"
    style="color: transparent;"
    width="150"
  />
</div>
`;
